test cases/frameworks/6 gettext/data3/meson.build

```diff
@@ -0,0 +1,36 @@
+
+if itstool.found()
+
+
+mi_translated = i18n.itstool_join(
+  input:  'com.mesonbuild.test.intlprog.metainfo.xml',
+  output: 'com.mesonbuild.test.intlprog.metainfo.xml',
+  mo_targets: mo_targets,
+  its_files: ['metainfo.its'],
+  install: true,
+  install_dir: join_paths(get_option('datadir'), 'metainfo'),
+)
+
+# older versions of itstool have a bug where ITS rules specified on the command-line
+# are not read when joining files. Since we don't install appstream in the Meson CI
+# environment, the to-be-tested entry will be untranslated and the test would fail, so
+# we just skip verification if the installed itstool is too old.
+r = run_command(itstool, '-v', check: true)
+itstool_v = r.stdout().strip().split()
+if itstool_v[1].version_compare('>=2.0.6')
+  grep_exe = find_program('grep')
```

Comment:
This is _not_ acceptable. Any use of shell tools is absolutely prohibited as all tests must run on a bog standard Windows `cmd.exe` without any unixy stuff. This should be a standalone Python script that reads the result file and validates it.